src/components/knapper/TilbakeKnapp.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { VenstreChevron } from 'nav-frontend-chevron';
+import styled from 'styled-components';
+import LocaleTekst from '../../language/LocaleTekst';
+import { Link } from 'react-router-dom';
+import { Normaltekst } from 'nav-frontend-typografi';
+
+const StyledTilbakeKnapp = styled.div`
+  padding: 1rem 0 1rem;
+  width: 75px;
+
+  .lenke {
+    color: #0067C5;
```

Comment:
Bruke variabel `@navBla`?

Reply:
(og på de andre stedene denne fargen er styla)

Reply:
Styled-components klarer å hente ut @navBla. Så for tilbakeknappen og slettknappen, vil det ikke funke hvis jeg bruker @navBla enn selve fargekoden. Men har endret den der vi brukes less filer til å style komponentene våre. :)